jiant/tasks/seq2seq.py

```diff
@@ -71,7 +72,10 @@ def get_data_iter(self, path):
                 if len(row) < 2 or not row[0] or not row[1]:
                     continue
                 src_sent = tokenize_and_truncate(self._tokenizer_name, row[0], self.max_seq_len)
-                tgt_sent = tokenize_and_truncate(self._tokenizer_name, row[2], self.max_seq_len)
+                if self.name == "seg_wix":
```

Comment:
Use a specific property of the task object, rather than the name. (It's fairly common to override name if you want to do MTL on multiple datasets for the same task.)

Reply:
Same for similar if statements below.

Reply:
Done.